sig-release/annual-report-2022.md

```diff
@@ -2,76 +2,105 @@
 
 ## Current initiatives
 
-1. What work did the SIG do this year that should be highlighted?
+SIG Release has a [roadmap](https://github.com/kubernetes/sig-release/blob/master/roadmap.md) that captures high level initiatives that the SIG is working toward, with specific information captured in our [project board](https://github.com/orgs/kubernetes/projects/30). 
 
-   -
-   -
-   -
+1. What work did the SIG do this year that should be highlighted?
```

Comment:
Do we want to add https://github.com/kubernetes/enhancements/issues/3027 and https://github.com/kubernetes/enhancements/issues/2853 as well?